konduit-cli/Readme.md

```diff
@@ -0,0 +1,16 @@
+# CLI for Konduit
+
+using the Konduit CLI you can easily create high performance model servers
+from simple python scripts with arbitrarily complex pre/post processing.
```

Comment:
is this for python only? if so, it might make sense to build it directly with `click` in Python, to attract the community to participate in fixes or extensions.

Reply:
Not only python. It’ll work for anything that we can configure Konduit with. I wanted to make this a native CLI app so that it could be used without relying on the user having to have anything installed. I think we can still implement a plugin system but I think most things would be more useful in the server side... need to think about that more.